Bio/Seq.py

```diff
@@ -315,6 +320,17 @@ def __repr__(self):
         data = self._data
         if isinstance(data, _UndefinedSequenceData):
             return f"Seq(None, length={len(self)})"
+        if isinstance(data, _PartiallyDefinedSequenceData):
+            d = {}
+            for position, seq in data._data.items():
+                if len(seq) > 60:
+                    start = seq[:54].decode("ASCII")
+                    end = seq[-3:].decode("ASCII")
+                    seq = "%s...%s" % (start, end)
+                else:
+                    seq = seq.decode("ASCII")
+                d[position] = seq
+            return "Seq(%r, length=%d)" % (d, len(self))
```

Comment:
CodeCov is pointing out that the above new code of partially defined sequences does not get tested when a fragment is over 60 letters.

Reply:
Added a test